scijava-function/src/main/java/org/scijava/function/Computers.java

```diff
@@ -119,7 +230,8 @@ private Computers() {
 	 * @throws NullPointerException If {@code c} is {@code null}.
 	 */
 	public static boolean isComputer(Class<?> c) {
-		return ALL_COMPUTERS.containsValue(c);
+		return ALL_COMPUTERS.values().stream().anyMatch(map -> map.containsValue(
+			c));
```

Comment:
I think using a `Set` data structure for this would be nicer. You could call it `COMPUTERS` or `COMPUTER_SET`, and then rename `ALL_COMPUTERS` to `COMPUTERS_BY_ARITY` or some such, for clarity.